internal/ls/findallreferencesexport_test.go

```diff
@@ -0,0 +1,31 @@
+package ls
+
+import (
+	"github.com/microsoft/typescript-go/internal/astnav"
+	"github.com/microsoft/typescript-go/internal/core"
+	"github.com/microsoft/typescript-go/internal/lsp/lsproto"
+)
+
+func (l *LanguageService) GetExpectedReferenceFromMarker(fileName string, pos int) *lsproto.Location {
```

Comment:
This is unusual but I guess it's ok if it's only temporary until we have fourslash.

Reply:
Yes, the testing exports is intended to be temporary, at least until we get fourslash more fleshed out